include/nvector/nvector_openmp.h

```diff
@@ -71,128 +72,227 @@ typedef struct _N_VectorContent_OpenMP *N_VectorContent_OpenMP;
  * -----------------------------------------------------------------
  */
 
-#define NV_CONTENT_OMP(v)  ( (N_VectorContent_OpenMP)(v->content) )
+#define NV_CONTENT_OMP(v) ((N_VectorContent_OpenMP)(v->content))
 
-#define NV_LENGTH_OMP(v)   ( NV_CONTENT_OMP(v)->length )
+#define NV_LENGTH_OMP(v) (NV_CONTENT_OMP(v)->length)
 
-#define NV_NUM_THREADS_OMP(v)   ( NV_CONTENT_OMP(v)->num_threads )
+#define NV_NUM_THREADS_OMP(v) (NV_CONTENT_OMP(v)->num_threads)
 
-#define NV_OWN_DATA_OMP(v) ( NV_CONTENT_OMP(v)->own_data )
+#define NV_OWN_DATA_OMP(v) (NV_CONTENT_OMP(v)->own_data)
 
-#define NV_DATA_OMP(v)     ( NV_CONTENT_OMP(v)->data )
+#define NV_DATA_OMP(v) (NV_CONTENT_OMP(v)->data)
 
-#define NV_Ith_OMP(v,i)    ( NV_DATA_OMP(v)[i] )
+#define NV_Ith_OMP(v, i) (NV_DATA_OMP(v)[i])
 
 /*
  * -----------------------------------------------------------------
  * Functions exported by nvector_openmp
  * -----------------------------------------------------------------
  */
 
-SUNDIALS_EXPORT N_Vector N_VNew_OpenMP(sunindextype vec_length, int num_threads, SUNContext sunctx);
+SUNDIALS_EXPORT
+N_Vector N_VNew_OpenMP(sunindextype vec_length, int num_threads,
+                       SUNContext sunctx);
 
-SUNDIALS_EXPORT N_Vector N_VNewEmpty_OpenMP(sunindextype vec_length, int num_threads, SUNContext sunctx);
+SUNDIALS_EXPORT
+N_Vector N_VNewEmpty_OpenMP(sunindextype vec_length, int num_threads,
+                            SUNContext sunctx);
 
-SUNDIALS_EXPORT N_Vector N_VMake_OpenMP(sunindextype vec_length, realtype *v_data,
-                                        int num_threads, SUNContext sunctx);
+SUNDIALS_EXPORT
+N_Vector N_VMake_OpenMP(sunindextype vec_length, realtype* v_data,
+                        int num_threads, SUNContext sunctx);
 
-SUNDIALS_EXPORT sunindextype N_VGetLength_OpenMP(N_Vector v);
+SUNDIALS_EXPORT
+sunindextype N_VGetLength_OpenMP(N_Vector v);
 
-SUNDIALS_EXPORT void N_VPrint_OpenMP(N_Vector v);
+SUNDIALS_EXPORT
+void N_VPrint_OpenMP(N_Vector v);
 
-SUNDIALS_EXPORT void N_VPrintFile_OpenMP(N_Vector v, FILE *outfile);
+SUNDIALS_EXPORT
+void N_VPrintFile_OpenMP(N_Vector v, FILE* outfile);
 
+SUNDIALS_EXPORT
+N_Vector_ID N_VGetVectorID_OpenMP(N_Vector v);
```

Comment:
Yeah, youre right that it doesnt usually matter because of the use of the generic functions. I didnt change them all in the PR because it didn't seem worth the effort yet for that reason.